lib/utils.js

```diff
@@ -158,6 +160,47 @@ exports.trimUnderscores = function(name) {
 };
 
 /**
+ * Whether or not the given path is relative
+ * @param  {String}  path
+ * @return {Boolean}
+ */
+exports.isRelativePath = function(path) {
```

Comment:
I think the existing solution is much clearer and more performant. Any edge cases that your method accounts for?